pkgs/applications/misc/k40-whisperer/default.nix

```diff
@@ -0,0 +1,64 @@
+{ stdenv
+, makeWrapper
+, writeText
+, python3
+, fetchzip
+, lib
+}:
+
+let
+  pythonEnv = python3.withPackages (ps: with ps; [
+    lxml
+    pyusb
+    pillow
+    pyclipper
+    tkinter
+  ]);
+
+  udevRule = writeText "k40-whisperer.rules" ''
+    SUBSYSTEM=="usb", ATTRS{idVendor}=="1a86", ATTRS{idProduct}=="5512", ENV{DEVTYPE}=="usb_device", MODE="0664", GROUP="users"
```

Comment:
Would it be prudent to create a dedicated group for this device? If so, we could also use `programs.k40-whisperer.users`.

Reply:
Should a package depend on a config module?

Reply:
Configurable via argument, defaulting to `k40`.